### PR TITLE
Document how to add individual story description, see #8527 #10823

### PR DESCRIPTION
**Pull request by vidhill**
Issue:

## What I did
Update docs to show how to add individual story description in `CSF` see #8527 #10823

\- I spent a silly amount of time trying to find this piece of info, I want to save the next person the trouble

## How to test

- Is this testable with Jest or Chromatic screenshots? no
- Does this need an update to the documentation? n/a

### CODE REVIEW

**Review by shilman — CHANGES_REQUESTED** [review]
This is actually the deprecated way of doing it. The recommended way is here:

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#docs-description-parameter

**Comment by vidhill** [comment]
Oh, I did not find that method either, I just updated my example, or is it documented elsewhere?

**Comment by shilman** [comment]
@vidhill yes, documented here https://storybook.js.org/docs/react/writing-docs/doc-blocks#description

**Comment by vidhill** [comment]
Shall I close/cancel this PR so?

File: addons/docs/docs/recipes.md [hunk 1]
```diff
@@ -336,6 +337,24 @@ import { theme } from '../path/to/theme'
 Rest of your file...
 ```
 
+## Add description to individual stories
+
+Add `storyDescription` to `parameters.docs`
```
Comment (shilman) [inline addons/docs/docs/recipes.md:342]:
```suggestion
Add `story` to `docs.description` parameter
```

**Review by shilman — APPROVED** [review]
LGTM! Merging this altho it overlaps with the official docs

**Pull request merged**